Packs/CommonScripts/Scripts/IPToHost/IPToHost.py

```diff
@@ -0,0 +1,40 @@
+import socket
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def ip_to_host(ip: str) -> CommandResults:
+    host_info = socket.gethostbyaddr(ip)
+
+    if not host_info:
+        raise DemistoException('No results were found for the given value.')
```

Comment:
This is a change in behavior (BC), in previous versions we did not fail the script in cases of no results. Why have we changed this?